packages/desktop-client/src/components/autocomplete/Autocomplete.tsx

```diff
@@ -239,6 +247,26 @@ function SingleAutocomplete<T extends Item>({
   );
   const [highlightedIndex, setHighlightedIndex] = useState(null);
   const [isOpen, setIsOpen] = useState(embedded);
+  const { isNarrowWidth } = useResponsive();
+  const narrowInputStyle = isNarrowWidth
+    ? {
+        ...styles.mobileMenuItem,
+      }
+    : {};
+
+  inputProps = {
+    ...inputProps,
+    style: {
+      ...narrowInputStyle,
+      ...inputProps.style,
+    },
+  };
+
+  useEffect(() => {
+    if (!isOpen) {
+      onClose?.();
+    }
+  }, [isOpen, onClose]);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is `onClose` called if the component is mounted with `isOpen = false`? Is that correct? Could it lead to some issues?